RELEASE_NOTES

```diff
@@ -3,8 +3,45 @@ For more detailed information, please see the git log.
 
 These release notes can also be consulted at http://easybuild.readthedocs.org/en/latest/Release_notes.html.
 
-The latest version of easybuild-easyconfig provides 7,566 easyconfig files, for 1,293 different software packages,
-25 different (compiler) toolchains, 13 software bundles and 1 meta-package.
+The latest version of easybuild-easyconfig provides 7,750 easyconfig files, for 1,307 different software packages,
+26 different (compiler) toolchains, 13 software bundles and 1 meta-package.
+
+v3.4.1 (October 16th 2017)
```

Comment:
17th?

Reply:
I'll correct that when making the PR to master, please proceed with merging.